src/splat/segtypes/common/textbin.py

```diff
@@ -18,6 +41,9 @@ def get_section_flags(self) -> Optional[str]:
         return "ax"
 
     def out_path(self) -> Optional[Path]:
+        if self.use_src_path:
```

Comment:
I think we should probably make this a generic setting for any segment, and I'd prefer to name it something like "path_relative_to" and allow "src", "data", etc. and have it inherit from parent segments/  global scope. but we can do that later